.github/workflows/windows.yml

```diff
@@ -100,6 +123,12 @@ jobs:
     runs-on: windows-2022
     needs: build-windows
     steps:
+      - name: Configure pagefile
+        uses: al-cheb/configure-pagefile-action@v1.3
+        with:
+          minimum-size: 8GB
+          maximum-size: 24GB
+          disk-root: "C:"
```

Comment:
Was the page file too small? Or why did you add this?

Reply:
The idea is to give more virtual memory via pagefile to the Windows build instances, since they have unused space on C: drive. See https://github.com/easimon/maximize-build-space#how-it-works for the inspiration.

Did it actually help? I cannot say really.